logging/cluster-logging-release-notes.adoc

```diff
@@ -47,35 +47,82 @@ To see these metrics, open the *Administrator* perspective in the {product-title
+
+* Previously, the OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator did not have the annotation that was required for them to appear in the {product-title} web console list of operators that can run in a disconnected environment. This update adds the `operators.openshift.io/infrastructure-features: '["Disconnected"]'` annotation to these two operators so they appear in the list of operators that run in disconnected environments.
+link:https://issues.redhat.com/browse/LOG-1420[LOG-1420]
+
 * link:https://issues.redhat.com/browse/LOG-1440[LOG-1440] "BZ#1966561 - OLM bug workaround for workload partitioning (PR#1042)"
+
 * link:https://issues.redhat.com/browse/LOG-1499[LOG-1499] "release-5.2 - Error 'com.fasterxml.jackson.core.JsonParseException: Invalid UTF-8 start byte 0x92' in Elasticsearch/Fluentd logs"
+
 * link:https://issues.redhat.com/browse/LOG-1567[LOG-1567] "Use correct variable for nextIndex"
-* link:https://issues.redhat.com/browse/LOG-1446[LOG-1446] "kibana-proxy CrashLoopBackoff with error Invalid configuration cookie_secret must be 16, 24, or 32 bytes to create an AES cipher"
+
+* Before this update, the kibana-proxy Pod sometimes entered the CrashLoopBackoff state and logged the following message "Invalid configuration: cookie_secret must be 16, 24, or 32 bytes to create an AES cipher when pass_access_token == true or cookie_refresh != 0, but is 29 bytes." The exact actual number of bytes could vary. With this update, generation of the Kibana session secret has been corrected, and the kibana-proxy Pod no longer crash loops due to this error.
+link:https://issues.redhat.com/browse/LOG-1446[LOG-1446]
+
 * link:https://issues.redhat.com/browse/LOG-1625[LOG-1625] "ds/fluentd is not created due to: 'system:serviceaccount:openshift-logging:cluster-logging-operator' cannot create resource 'securitycontextconstraints' in API group 'security.openshift.io' at the cluster scope"
-* link:https://issues.redhat.com/browse/LOG-1071[LOG-1071] "fluentd configuration posting all messages to its own log"
-* link:https://issues.redhat.com/browse/LOG-1276[LOG-1276] "Update Elasticsearch/kibana to use opendistro security plugin 2.10.5.1"
+
+* Before this update, the AWS CloudWatch fluentd plugin logged its AWS API calls to the fluentd log at all log levels, thus creating undue pressure on the resources of OCP nodes. With this update, the AWS CloudWatch fluentd plugin logs AWS API calls only at the "debug" and "trace" log levels. This way, at the default "warn" log level, fluentd does not consume extra OCP node resources. link:https://issues.redhat.com/browse/LOG-1071[LOG-1071]
+
+* Before this update, the Elasticsearch OpenDistro security plugin caused user index migrations to fail. This update resolves the issue by providing a newer version of the plugin. Now, index migrations proceed without errors.
+link:https://issues.redhat.com/browse/LOG-1276[LOG-1276]
```

Comment:
@periklis Please review and approve by replying to (or thumbs up emojiing) this comment.

Reply:
LGTM